src/components/Text/Text.tsx

```diff
@@ -33,28 +33,33 @@ interface TextProps
     inverted?: boolean;
     /**
      * Adjust color to indicate secondary information
+     * @deprecated
      */
     weak?: boolean;
+    /**
+     * Adjust color to indicate secondary information
+     */
+    secondary?: boolean;
```

Comment:
Question: How can we ensure the user knows that the `disabled` prop overrides the `weak` and `secondary` props? Maybe we can just add this to the documentation and the props table, so that the users are aware of it and don't have to look it up in the source code.

Reply:
I did two things:

- raised the `disabled` prop higher in the props table to implicitly show the precedence
- added an explicit text to the description